app/views/downloads/download_linux.html.erb

```diff
@@ -12,49 +12,74 @@
 
   <h3>Debian/Ubuntu</h3>
   <p>For the latest stable version for your release of Debian/Ubuntu</p>
-  <code># apt-get install git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># apt-get install git</pre>
   <p>For Ubuntu, this PPA provides the latest stable upstream Git version</p>
-  <code># add-apt-repository ppa:git-core/ppa</code>
-  <code># apt update; apt install git</code>
+  <pre style="height: 10px;overflow-y: hidden;padding-top: 0;margin-top: 0;"># add-apt-repository ppa:git-core/ppa</pre>
```

Comment:
instead of adding inline style here, can we add to the css file we have? it may be worth to add a custom class to avoid unwanted style changes in other parts of the site (not sure, we need to check)

Reply:
Sure, I will add the CSS in the HTML.